client/components/LoadingSpinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './LoadingSpinner.css';
+import PropTypes from 'prop-types';
+
+const LoadingSpinner = ({ percentage = null, className = null }) => {
+    return (
+        <div
+            className={
+                className
+                    ? `spinner-container ${className}`
+                    : 'spinner-container'
+            }
```

Comment:
A nice way to get this on one line would be 

```
className={`spinner-container ${className}`}
```

And change the default className to `className = ''`  on line 5.